### DISCUSSION TITLE
Handling of false recognitions

### DISCUSSION THREAD

**Discussion by oliverk71** [mcguirepr89/BirdNET-Pi#117] [Q&A] [answered]
Hi folks, 

I thought, it would be better to start a new discussion on this topic. If it was already discussed somewhere, please point me to it. 

Sometimes, I encounter false recognitions. Most of them are from noises in my neighbourhood, for example the neighbour's dog's barking is usually recognized as Ural Owl, which is highly unlikely, anyway. Great bittern is often recognized, but this is because right now the birdnet-pi is set up near a window, where a TV is behind. We do have great bittern 5 or 6 kilometers away, but I listened to the audios and it's just the TV behind the window. Of course, this may be optimized by a better position for the birdnet-pi and I will put it somewhere else, later. However.

My question is, how should those false recognitions be handled and how can they be prevented? 

Of course, I can login into the Maria db and manually delete them. But it would be better to delete often occuring false recognitions from the bird's list (labels.txt). If doing so, how should it be done? Just delete them and leave an empty line? I fear that birdnet-pi mixes the species if I do something wrong when editing the labels.txt.

As I heard, some use a labels.txt only with species which are likely to occur in their country. I would like to edit my labels.txt for my birdnet-pis specific position, especially regarding the often false recognized species, as the two mentioned above.

**Comment by nilspupils** [comment, marked as answer]
Hi, this has been discussed a couple of times and just yesterday came up again here: https://github.com/mcguirepr89/BirdNET-Pi/issues/115
It seems this has been implemented for testing just now by @mcguirepr89!
If you are interested in the custom species list for Europe, this is the revised version in german here: https://github.com/kahst/BirdNET-Lite/issues/4#issuecomment-995834655
